docs/pages/includes/permission-warning.mdx

```diff
@@ -1,4 +1,5 @@
-<Details title="Best practices for production security">
+<Admonition type="important">
+**Best practices for production security**
```

Comment:
When rendered, this Admonition looks large on the page, and is often longer than the main body of a section ([example](https://docs-7jarl72nu-goteleport.vercel.app/docs/admin-guides/api/getting-started/#step-13-create-a-user)). It's also difficult to tell that the first bolded line is a heading.

Reply:
For the first note, do you feel strongly about this? We could just use MD instead of tags, but we don't want folks to skim it. I like how this renders personally, as it's important information regardless of the sections sizes where it's used. I'll try out some other formatting to make the heading to be more bold.